### ISSUE TITLE
[$10][MSFT-61] [Register] Social media icon text foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2777]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Check the Social media icon text

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #3A3A3A. New contrast ratio: 4.55
- Passes AA, if you change text color to #757585. New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 2.49 (foreground color: #a3a3ae, background color: #ffffff, font size: 7.5pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F34

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096292 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096292 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096292 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu this PR covers it https://github.com/appirio-tech/accounts-app/pull/226

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097375 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097375 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@mishacucicea i solved this perfectly. kindly check here verification.

![2777](https://user-images.githubusercontent.com/28830240/62085116-6ca3bb00-b278-11e9-9022-5220ef067881.png)

**Comment by PrakashDurlabhji** [comment]
@mishacucicea here is PR https://github.com/appirio-tech/accounts-app/pull/279

**Comment by nithyaasworld** [comment]
@nkumar-topcoder I think this fix is not yet deployed. But what I am seeing is, the issue is no more an issue. It has good contrast for the social-media-texts.

Please find the screenshot below:
![image](https://user-images.githubusercontent.com/5114190/62184165-701f6b00-b37a-11e9-9da5-2bec38abf24d.png)

**Comment by nithyaasworld** [comment]
This issue seems to have already been resolved in Production too.

![image](https://user-images.githubusercontent.com/5114190/62184253-d60bf280-b37a-11e9-8d31-fdc7ab08cfc4.png)

**Comment by nithyaasworld** [comment]
@PrakashDurlabhji I am closing this issue as its no more an issue. Thanks for your effort!

**Comment by crazyk07** [comment]
Closing for payment

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097375<br/><br/>```This is an automated message for crazyk via Topcoder X```
